GeoDaCenter/chicago-environment-explorer
# feat: Added new variables - Air Temperature / Heat Index (Average) / Heat Index (Maximum)

## Problem
New variables have been submitted by @jluiuc: Average Temp + Average Heat Index

These need to be added to the ChiVes map

## Approach
* Added the new variables to the ChiVes map
* Recategorized / renamed variables according to shared spreadsheet

## How to Test
1. Navigate to https://deploy-preview-129--chicago-env-explorer.netlify.app/map
2. From the Variables list, choose "Air Temperature"
    * You should see the map update with the new values + colors + Legend
3. From the Variables list, choose "Heat Index (Average)"
    * You should see the map update with the new values + colors + Legend
4. From the Variables list, choose "Heat Index (Maximum)"
    * You should see the map update with the new values + colors + Legend

### Screenshots

![Screenshot 2024-03-11 at 2 31 07 PM](https://github.com/GeoDaCenter/chicago-environment-explorer/assets/1413653/73762507-d897-4c67-8499-11efd6c30f46)

![Screenshot 2024-03-11 at 2 31 21 PM](https://github.com/GeoDaCenter/chicago-environment-explorer/assets/1413653/3a2b5908-0634-48c8-9b88-b3b1d1790203)

![Screenshot 2024-03-11 at 2 31 28 PM](https://github.com/GeoDaCenter/chicago-environment-explorer/assets/1413653/68cf62d9-02a2-4dae-a026-01b18db22689)
